R/bayes.R

```diff
@@ -171,33 +171,6 @@ prep.step_lencode_bayes <- function(x, training, info = NULL, ...) {
   )
 }
 
-
-glm_coefs <- function(x, y, ...) {
```

Comment:
This function is present here: https://github.com/tidymodels/embed/blob/main/R/glm.R#L135-L166